src/nni_manager/training_service/remote_machine/remoteMachineData.ts

```diff
@@ -83,101 +84,71 @@ export class RemoteMachineTrialJobDetail implements TrialJobDetail {
     }
 }
 
-/**
- * The remote machine ssh client used for trial and gpu detector
- */
-export class SSHClient {
-    private readonly sshClient: Client;
-    private usedConnectionNumber: number; //count the connection number of every client
-    constructor(sshClient: Client, usedConnectionNumber: number) {
-        this.sshClient = sshClient;
-        this.usedConnectionNumber = usedConnectionNumber;
-    }
-
-    public get getSSHClientInstance(): Client {
-        return this.sshClient;
-    }
-
-    public get getUsedConnectionNumber(): number {
-        return this.usedConnectionNumber;
-    }
-
-    public addUsedConnectionNumber(): void {
-        this.usedConnectionNumber += 1;
-    }
-
-    public minusUsedConnectionNumber(): void {
-        this.usedConnectionNumber -= 1;
-    }
-}
-
 /**
  * The remote machine ssh client manager
  */
 export class SSHClientManager {
```

Comment:
class ShellExecutorManager ?

Reply:
updated to ExecutorManager, it may extend to more kinds of executor.